ansible/roles/etcd/tasks/install.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Install etcd
+  action: "{{ ansible_pkg_mgr }}"
+  args:
+        name: etcd
+        state: latest
+  notify:
+    - restart etcd
+  when: not is_atomic and ansible_distribution != "Ubuntu" and not is_coreos
+
+- name: Force etcd_source_type to github when packages are not available
+  set_fact:
+    etcd_source_type: "github-release"
```

Comment:
We use `source_type` [numerous times] (https://github.com/kubernetes/contrib/search?utf8=%E2%9C%93&q=source_type) in the project. I'll use that for now. If we need more granularity down the road each instance of `source_type` can be prepended by its service or package name.